lib/diagnostic.js

```diff
@@ -40,7 +50,9 @@ const hasNone = function (array, callback) {
 // you may use any feature of JavaScript
 // to solve this problem
 const last = function (array, callback) {
-
+  const newArray = array.forEach(callback)
```

Comment:
Right idea, but remember that `forEach` does not automatically create a new array for you. `Map` would probably be a better option.